docs/getting-started/croeseid-testnet.md

```diff
@@ -280,7 +299,49 @@ confirm transaction before signing and broadcasting [y/N]: y
 
 :::
 
+### Step 4-3. `tx staking ` - Staking operations
+
+Staking operations involve the interaction between an address and a validator. It allows you to create a validator and lock/unlocking funds for staking purposes.
+
+#### **Delegate you funds to a validator** [`tx staking delegate <validator-addr> <amount>`]
+
+To bond funds for staking, you can deposit funds (an unspent transaction) to a _staking_ address by the `Deposit` operation.
```

Comment:
```suggestion
To bond funds for staking, you can delegate funds  to a validator by the `delegate` command
```

Sorry I think there is some  thaler concept left in here: 
https://github.com/crypto-com/chain-docs/blob/master/docs/wallets/cli.md#delegate-you-funds-to-a-validator-tx-staking-delegate-validator-addr-amount